lms/templates/verify_student/image_input.underscore

```diff
@@ -0,0 +1,2 @@
+<img class="preview" alt="<%- gettext("Preview of uploaded image") %>"/>
+<input class="image-upload" type="file" accept="image/*;capture=camera">
```

Comment:
@cptvitamin This is element I'm most concerned about for a11y.  Is there anything I can add to make this more accessible, especially on a mobile browser?